assets/src/edit-story/components/header/titleMissingDialog.js

```diff
@@ -40,42 +40,33 @@ const Paragraph = styled.p`
 `;
 
 const GUIDELINES =
-  'https://drive.google.com/corp/drive/folders/1dqPtjNTYN7OxTngWAYhy8zn8X3EuRTGJ';
+  'https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#title';
```

Comment:
Since the AMP documentation is available in different languages, let's make sure this URL is translatable (using `__()`).

Reply:
Fixed in 18c11d709.